amgut/db/initialize.sql

```diff
@@ -1,4 +1,4 @@
-SET search_path TO ag, public;
+SET search_path TO ag, public, barcodes;
```

Comment:
probably best to put `barcodes` before public; these namespaces are searched in the order specified